hackathon_site/registration/admin.py

```diff
@@ -23,8 +26,47 @@ def get_queryset(self, request):
         return super().get_queryset(request).prefetch_related("applications")
 
 
+class ApplicationResource(resources.ModelResource):
+    class Meta:
+        model = Application
+
+        fields = (
+            "user__first_name",
+            "user__last_name",
+            "user__email",
+            "team",
```

Comment:
Maybe add team code too? The exporter you wrote for makeuoft seems to use call `__str__` on the foreign key so we actually get the team code, but this library just puts the foreign key ID. Could probably just replace with `team__code` or whatever it is, since nobody will need the actual team ID on export.